rinex/src/lib.rs

```diff
@@ -2815,82 +2868,6 @@ impl Decimate for Rinex {
     }
 }
 
-#[cfg(feature = "obs")]
-use crate::observation::Observation;
-
-#[cfg(feature = "obs")]
-#[cfg_attr(docrs, doc(cfg(feature = "obs")))]
-impl Observation for Rinex {
-    fn min(&self) -> (Option<f64>, HashMap<Sv, HashMap<Observable, f64>>) {
-        if let Some(r) = self.record.as_obs() {
-            r.min()
-        } else {
-            (None, HashMap::new())
-        }
-    }
-    fn min_observable(&self) -> HashMap<Observable, f64> {
-        if let Some(r) = self.record.as_obs() {
-            r.min_observable()
-        } else if let Some(r) = self.record.as_meteo() {
-            r.min_observable()
-        } else {
-            HashMap::new()
-        }
-    }
-    fn max(&self) -> (Option<f64>, HashMap<Sv, HashMap<Observable, f64>>) {
-        if let Some(r) = self.record.as_obs() {
-            r.max()
-        } else {
-            (None, HashMap::new())
-        }
-    }
-    fn max_observable(&self) -> HashMap<Observable, f64> {
-        if let Some(r) = self.record.as_obs() {
-            r.max_observable()
-        } else if let Some(r) = self.record.as_meteo() {
-            r.max_observable()
-        } else {
-            HashMap::new()
-        }
-    }
-    fn mean(&self) -> (Option<f64>, HashMap<Sv, HashMap<Observable, f64>>) {
-        if let Some(r) = self.record.as_obs() {
-            r.mean()
-        } else {
-            (None, HashMap::new())
-        }
-    }
-    fn mean_observable(&self) -> HashMap<Observable, f64> {
-        if let Some(r) = self.record.as_obs() {
-            r.mean_observable()
-        } else if let Some(r) = self.record.as_meteo() {
-            r.mean_observable()
-        } else {
-            HashMap::new()
-        }
-    }
-    fn std_dev(&self) -> (Option<f64>, HashMap<Sv, HashMap<Observable, f64>>) {
-        if let Some(r) = self.record.as_obs() {
-            r.std_dev()
-        } else {
-            (None, HashMap::new())
-        }
-    }
-    fn std_dev_observable(&self) -> HashMap<Observable, f64> {
-        HashMap::new()
-    }
-    fn std_var(&self) -> (Option<f64>, HashMap<Sv, HashMap<Observable, f64>>) {
-        if let Some(r) = self.record.as_obs() {
-            r.std_var()
-        } else {
-            (None, HashMap::new())
-        }
-    }
-    fn std_var_observable(&self) -> HashMap<Observable, f64> {
-        HashMap::new()
-    }
-}
-
```

Comment:
The previous "Observation" trait was developed to evaluate statistics on Observations, for QC analysis.
I don't think i was a good idea to do that.
Now that we have easier methods to navigate the data, it's easy for the library user to use `statrs` to compute whatever statistics they want.
In QC, where we need to evaluate statistics : simply use `statrs`